src/libraries/System.Text.Json/src/System/Text/Json/JsonPropertyDictionary.KeyCollection.cs

```diff
@@ -8,13 +8,6 @@ namespace System.Text.Json
 {
     internal sealed partial class JsonPropertyDictionary<T>
```

Comment:
@stephentoub I have a local change renaming this to `OrderedDictionary<TKey, TValue>`, effectively converting it to a polyfill. I figured I'd push this change later since it would result in a substantially noisier diff.